### ISSUE TITLE
[VISUAL] Skyboxes are culled when looking away with a high enough FOV

### ISSUE THREAD

**Issue by HoneyfrostStudio** [mapbase-source/source-sdk-2013#305]
### Describe the bug
When the FOV exceeds 85, the skybox gets culled on the sides, revealing the void.

### Steps to reproduce
1. Modify the code to change minimum and maximum FOV values
2. To better notice the issue, set your FOV to an absurdly high value, like 150.
3. Observe

### Fix
There's a fix for that; ficool2 talked about it a while ago on mapbase discord server:
https://discord.com/channels/548979532284297237/564519048503885845/782284538009092137

### Media
![Capture d'écran 2024-06-22 233911](https://github.com/mapbase-source/source-sdk-2013/assets/157644886/b5f153c6-5eb0-49f0-9949-87854d794977)

**Comment by meatspace** [comment]
yeah it can be fixed like this if blixi so chooses: https://github.com/momentum-mod/game/pull/612

**Comment by Cvoxalury** [comment]
Hopefully Blixibon doesn't choose it, as it is truly insane - in essense guessing and patching what is _assumed_ to be the right thing and _assuming_ it won't cause any issues down the line. This kind of work is just too risky for Mapbase, like other proposed operations with memory hacks and patching things on the fly.

**Comment by WhiteRedDragons** [comment]
In my opinion this is pretty much a non-issue
The FOV Values are capped for a reason, you have to modify the safe min/max values to cause this to begin with

**Comment by Blixibon** [comment]
I agree that this isn't worth changing for now given the risks attached to trying to fix it.
